bool bLeftArrow
bool bRightArrow
bool bUpArrow
bool bDownArrow
bool bA
bool bB
bool bC
bool bD
bool bE
bool bF
bool bG
bool bH
bool bI
bool bJ
bool bK
bool bL
bool bM
bool bN
bool bO
bool bP
bool bQ
bool bR
bool bS
bool bT
bool bU
bool bV
bool bW
bool bX
bool bY
bool bZ
bool b1
bool b2
bool b3
bool b4
bool b5
bool b6
bool b7
bool b8
bool b9
bool b0
